dev/integration_tests/android_views/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: android_views
+description: An integration test for embedded Android views
+version: 1.0.0+1
+
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_driver:
+    sdk: flutter
+  path_provider: ^0.4.1
+  collection: ^1.14.6
+  assets_for_android_views:
+    path: ../../../bin/cache/pkg/goldens/dev/integration_tests/assets_for_android_views
```

Comment:
reminder to edit this before submitting

Reply:
Clarified f2f - we don't publish this to pub, we're actually fetching the goldens in the driver test and reading from this path.